### DISCUSSION TITLE
About achievement “YOLO”

### DISCUSSION THREAD

**Discussion by zynowhere** [community/community#18327] [Projects and Issues]
Earned by “Merged pull request without code review”

How “without code review”？

**Comment by ManuelLoaizaV** [comment]
Some months ago, I opened a hot fix pull request in a project that a friend and I have been working on since last year.
We usually [request each other for a code review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) as a best practice even though it was not a must like in a workplace.

![image](https://user-images.githubusercontent.com/19366788/173174408-0625d688-f4d3-4a5f-a774-3e4bd3a0a413.png)

I had planned to create a new release that day and needed to merge that pull request which had a minor change related to GitHub Worflows. However, my friend messaged me some hours before that he was going to be without internet connection until he returned to the city because his family including him were going on a trip to the mountains in Peru for at least two days.

![image](https://user-images.githubusercontent.com/19366788/173174963-bb17262f-610b-4b02-a1b5-097c48530804.png)

So I decided to merge that PR without his approval in order to deploy and create the release.

**Comment by mr-cheffy** [comment]
I think it is when you just merge the pull request without even looking at the code they changed

**Comment by HUMBLEGROUND** [comment]
You want it? You merge it.

**Comment by melvincwng** [comment]
@zynowhere , to get "YOLO" achievement badge, try doing the following:

1. Create a PR, add a friend as a reviewer, but merge the PR without your friend's approval
2. Alternatively, create a PR in your code repository, and self-merge directly
3. Ensure your code repository is public. 

Hope the following steps work for you

**Comment by ATLUSio** [comment]
You create a PR with reviewers, but you don't wait for the reviewers to accept the PR.\
The "without code review" is the lack of approvals (review) from the listed reviewers.
